gems/sorbet-runtime/lib/types/_types.rb

```diff
@@ -155,6 +155,14 @@ def self.assert_type!(value, type, checked: true)
     Private::Casts.cast(value, type, cast_method: "T.assert_type!")
   end
 
+  # Tells the typechecker to ensure that `value` is of type `type` (if not, the typechecker will
+  # fail). Use this for debugging typechecking errors, or to ensure that type information is
+  # statically known and being checked appropriately. If `checked` is true, raises an exception at
+  # runtime if the value doesn't match the type.
```

Comment:
This comment looks out of date?

Reply:
Comment updated to be correct now. I've also added a bit to the docs, since I had forgotten to do that.